package.json

```diff
@@ -46,6 +46,7 @@
     "@storybook/addon-viewport": "^6.0.21",
     "@storybook/react": "^6.0.21",
     "@types/react": "^17.0.0",
+    "@types/react-burger-menu": "^2.8.0",
```

Comment:
How is this package needed when a type was created for the burger menu already?

Reply:
I'm not exactly sure what you're asking here, but just to be clear, we're using the `react-burger-menu` library to implement our own `<BurgerMenu>` component, which just wraps the `react-burger-menu`'s `push` component (renamed to `Menu` in the import statement). We need type declaration on both the library we're using as well as our own component, since both are instantiated in our code.